scripts/mkcloud

```diff
@@ -292,82 +292,84 @@ function error_exit
 function sshrun
 {
     cat > $mkcconf <<EOF
```

Comment:
If they're indented with tabs, you can use <<- and bash will remove the leading tabs

Reply:
Thanks! Do you think that would look more readable? I'll change it if you think it would enhance it.